README.md

```diff
@@ -48,6 +48,9 @@ In all of the hacktoberfest excitement, many companies want to join the celebrat
 | Uno | ![Shirt](icons/tshirt-greyscale.png) ![Swag](icons/swag-greyscale.png) | "anyone who contributes a pull request to the Uno Platform on GitHub between October 1 and October 31 will receive a Uno Platform T-shirt plus some limited edition swag." | <https://platform.uno/uno-is-joining-hacktoberfest-2019/> (Verified 2019) |
 | Valor | ![Stickers](icons/stickers-greyscale.png) ![Swag](icons/swag-greyscale.png) | 1 PR: pen and sticker. 3 PRs: notepad or wooden phone stand. 5 PRs: pen, sticker, notepad, and a wooden phone stand. | <https://valor-software.com/articles/hacktoberfest-2019-is-coming-and-ngx-bootstrap-strives-for-your-attention.html> (Verified 2019) |
 
+| Devfolio | ![Stickers](icons/stickers-greyscale.png) ![Swag](icons/stickers-greyscale.png) | PR: T-shirt, stickers. | <https://devfolio.co/blog/hacktoberfest-2019-devfolio/> (Verified 2019) |
```

Comment:
You use the stickers image file twice here, so please change one to the t-shirt image. :shirt:

Reply:
(and the image description should reflect the actual image)